api/doris/v1/types.go

```diff
@@ -95,11 +95,12 @@ type CnSpec struct {
 
 // BrokerSpec describes a template for creating copies of a broker software service, if deploy broker we recommend you add affinity for deploy with be pod.
 type BrokerSpec struct {
-	//expose the cn listen ports
-	Service ExportService `json:"service,omitempty"`
-
 	//the foundation spec for creating cn software services.
-	BaseSpec `json:"baseSpec,omitempty"`
+	//BaseSpec `json:"baseSpec,omitempty"`
+	BaseSpec `json:",inline"`
+
+	// Enable affinity configuration switch, default:false
```

Comment:
enable affinity with be , if kickoff affinity, the operator will set affinity on broker with be, the affinity is preferred not required. when the user custom affinity the switch does not take effect anymore.